pages/linux/lxc.md

```diff
@@ -0,0 +1,32 @@
+# lxc
+
+> Manage Linux containers using the lxd REST API.
+> 'remote:' refers to the alias for a remote server (i.e. 'ubuntu:') and when omitted will apply locally.
```

Comment:
I don't think the `(i.e. 'ubuntu:')` is particularly elucidative here. In fact, the whole sentence is longer than I'd be comfortable with. Maybe I'd write this as:

    Any container names or patterns can be prefixed with the name of a remote server.

Does that sound OK to you guys?